src/highrl/envs/teacher_env.py

```diff
@@ -27,6 +27,7 @@
 from prettytable import PrettyTable
 import argparse
 from os import path
```

Comment:
`prettytable` library import should be place after builtin import `os` and `argparse`.

Reply:
Done